python/feathub/processors/spark/ast_evaluator/tests/test_spark_ast_evaluator.py

```diff
@@ -82,7 +82,7 @@ def test_func_call(self):
         )
 
     def test_cast(self):
-        self.assertEqual("CAST('59' AS BYTES)", self._eval('cast("59" AS bytes)'))
+        self.assertEqual("TO_BINARY('59', 'UTF-8')", self._eval('cast("59" AS bytes)'))
```

Comment:
Why do we need to replace the existing line?

Can you explain why supporting parquet would break the results of executing "CAST('59' AS BYTES)"?

Reply:
The result of executing "CAST('59' AS BYTES)" is not broken. It is just that the actual execution of this command in Spark SQL has not been covered in tests yet. In `test_parquet_all_types`, it is revealed that `CAST('59' AS BYTES)` is not supported in Spark.

I just found that `CAST('59' AS BINARY)` is a better solution than `TO_BINARY('59', 'UTF-8')`. I'll update the test here in this way.